### DISCUSSION TITLE
Preview Links Question

### DISCUSSION THREAD

**Discussion by bking124** [quarto-dev/quarto-web#282] [Q&A]
Not sure if this is the right forum, but I had a question about previewing links with Quarto presentations.  It seems that a lot of websites show "refused to connect" when trying to use the preview-link="true" option. For example, I was trying to preview a Github repo but that refuses to connect.  After some digging, I guess this has something to do with website security and websites not wanting to be put into an iframe, but I'm very naive in this setting so not 100% sure.

Is there any way to get around this and preview sites like Github?

The source code for an example slide would be:
```
## Check Out Github

::: {style="text-align: center; margin-top: 1em"}
[Look at Github](https://github.com/){preview-link="true" style="text-align: center"}
:::
```
And when I try to click on that link in the presentation I see this:
![image](https://user-images.githubusercontent.com/18384932/181357818-63cfdecb-71cf-489b-951a-16c59673647c.png)

**Comment by andrewpbray** [comment]
I don't have an authoritative answer for you, but looked into this awhile ago and came to a similar conclusion as you. I was trying to put RStudio server in an iframe in fact, and found that there's an issue with a particular parameter in the header of the HTML file that prevents iframe embedding. I dug a bit but didn't find a workaround.
